### PR TITLE
BE-012 Docker compose1.5 (Anton Rudkovskyi)

### PR DESCRIPTION
**Pull request by rudik-byte**
Edit docker_compose
Added new service for docker_compose

### CODE REVIEW

**Review by el-dub — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  backend:
+    build: .
+    ports:
+      - 8080:8080
+    restart: always    # при несанкционированных падениях апки, будет произведен рестарт
+    links:
```
Comment (SergeyKoval) [inline docker-compose.yml]: Links is deprecated now, use depends_on

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

**Comment by SergeyKoval** [comment]
Your backend service is not using db. Please connect them and review, that it starts without any issues

File: docker-compose.yml (hunk 1 above)
Comment (rudik-byte) [reply docker-compose.yml]: Done

**Review by SerhiiPakulia — DISMISSED** [review]

**Review by el-dub — APPROVED** [review]

**Review by el-dub — DISMISSED** [review]

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+spring:
+  jpa:
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+    datasource:
+      driver-class-name: org.postgresql.Driver
+      url:${POSTGRES_DB_URL}
```
Comment (SergeyKoval) [inline src/main/resources/application.yml]: It is bad approach. Your application yml doesn't know about docker compose environment variables. Specify url here directly

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  backend:
```
Comment (SergeyKoval) [inline docker-compose.yml:3]: You should pass correct environment variables for the backend service. It should be at least url (which will be different from your application.yml specified. Read about using another docker compose service name as part of the url

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

**Pull request closed without merging**